### PR TITLE
Editing a small section of the wiki about systray

### PR DESCRIPTION
**Pull request by AtariBaby**
creating pull request per PearsonFlyer and BakerBoy448

### CODE REVIEW

File: radarr/installation/multiple-instances.md [hunk 1]
```diff
@@ -94,18 +94,28 @@ separate locations. {.is-warning}
...
-- Create a new folder for Radarr-4K's configuration files. Most use a similar place such as
-    `C:\ProgramData\Radarr-4K`
-- Right click and Create New Shortcut
-- Path: `C:\ProgramData\Radarr\bin\Radarr.exe /data=C:\ProgramData\Radarr-4K`
-- Give the shortcut a unique name such as `Radarr-4K` and finish the wizard.
-- Double click the new shortcut to run and test.
-- Continue to [Configuring Radarr-4k](#windows-multi-config-second)
+You must have Radarr already installed
+
+1. In C:/ProgramData, create a new folder for Radarr-4K's configuration files. Most use a similar place such as C:\ProgramData\Radarr-4K
```
Comment (bakerboy448) [inline radarr/installation/multiple-instances.md:108]:
```suggestion
1. In `C:\ProgramData`, create a new folder for Radarr-4K's configuration files. Most use a similar place such as `C:\ProgramData\Radarr-4K`
```

File: radarr/installation/multiple-instances.md [hunk 2]
```diff
@@ -94,18 +94,28 @@ separate locations. {.is-warning}
...
+
+2. Next we go to the startup folder. Go to “Run” and enter “%appdata%\Microsoft\Windows\Start Menu\Programs\Startup” and then OK.
```
Comment (bakerboy448) [inline radarr/installation/multiple-instances.md:110]:
```suggestion
2. Next we go to the startup folder. Go to “Run” and enter `%appdata%\Microsoft\Windows\Start Menu\Programs\Startup` and then OK.
```

File: radarr/installation/multiple-instances.md [hunk 3]
```diff
@@ -94,18 +94,28 @@ separate locations. {.is-warning}
...
+   
+3. Create the Radarr-4K Tray App.
+   Right click the Radarr shortcut, and Create New Shortcut.
+   Rename the shortcut “Radarr-4K.” Edit the target field of the new shortcut:
+
+   C:\ProgramData\Radarr\bin\Radarr.exe /data=C:\ProgramData\Radarr-4K
```
Comment (bakerboy448) [inline radarr/installation/multiple-instances.md:116]:
```suggestion
   `C:\ProgramData\Radarr\bin\Radarr.exe /data=C:\ProgramData\Radarr-4K`
```

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
couple edits and then if you can ensure all the steps are `1.` or `-`

thx

File: radarr/installation/multiple-instances.md [hunk 4]
```diff
@@ -53,9 +53,9 @@ This guide will show you how to run multiple instances of Radarr on Windows usin
 1. Open a Command Prompt administrator window. (To run as an admin,
     right click on the Command Prompt icon and choose “Run as
     administrator.”)
-1. If Radarr is running, stop the service by running `nssm stop Radarr`
+2. If Radarr is running, stop the service by running `nssm stop Radarr`
```
Comment (bakerboy448) [inline radarr/installation/multiple-instances.md:56]: Keep as 1. Wiki numbers it when rendering

**Comment by bakerboy448** [comment]
Will squash and merge for now and then you can PR an update

Something > nothing

File: radarr/installation/multiple-instances.md (hunk 4 above)
Comment (AtariBaby) [reply radarr/installation/multiple-instances.md:56]: Oh i had no idea!

**Pull request merged**